src/katatraining/collatzcojnjecturelength/Collatz.java

```diff
@@ -0,0 +1,17 @@
+package katatraining.collatzcojnjecturelength;
+
+public class Collatz {
+    public static long conjecture(long x) {
+        int i = 1;
+        do {
+            if (x % 2 == 0) {
+                x = x / 2;
+            } else {
+                x = 3 * x + 1;
+            }
+            i++;
+        }
+        while (x != 1);
```

Comment:
what will happen if there will be 0 or negative value as the conjecture method parameter? Will it work correctly?

Reply:
Only non-negative values. But I fixed for zero.